docs/content/kubernetes-primer.md

```diff
@@ -251,13 +251,13 @@ For example, it’s possible to accidentally configure two Kubernetes ingresses
 Below is a partial example of the input OPA sees when someone creates an ingress.  To avoid conflicts, we want to prevent two ingresses from having the same `request.object.spec.rules.host`.  If OPA has only this one ingress configuration it doesn't have enough information to make an allow/deny decision; it also needs the configurations for all of the existing ingresses.
 
 ```yaml
-apiVersion: admission.k8s.io/v1beta1
+apiVersion: admission.k8s.io/v1
 kind: AdmissionReview
 request:
   kind:
-    group: extensions
+    group: networking.k8s.io
```

Comment:
This is https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122, correct?

Looks like those aren't used in the policies in this document, so I guess we're OK.

Reply:
> This is https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122, correct?

Yes. I'm opening yet another PR because in this API version `pathType` is mandatory. Sorry, missed it.